.eslintrc.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
     'no-shadow': ERROR,
     'no-unused-expressions': ERROR,
     'no-unused-vars': [ERROR, {args: 'none'}],
+    'no-use-before-define': [ERROR, {functions: false, variables: false}],
```

Comment:
Naming of these options is confusing. It basically means "still allow to rely on function hoisting, and don't complain about variables *unless* they are in the same scope".

Reply:
> Naming of these options is confusing. It basically means "still allow to rely on function hoisting, and don't complain about variables unless they are in the same scope".

If `.eslintrc.js` is a `.js` would it make sense to put this explanation in a comment next to the added option?